File: cheese-api/cheese-api/Models/Cheese.cs
namespace cheese_api.Models
{
    public class Cheese
    {
        public long Id { get; set; }
        public string Name { get; set; }
        public string ImageUrl { get; set; }
        public string Color { get; set; }
        public decimal PricePerKilo { get; set; }
    }
}

File: cheese-front/src/models/Cheese.ts
export class Cheese {
    constructor(
        public id: number,
        public name: string,
        public imageUrl: string,
        public color: string,
        public pricePerKilo: number
    ) {
    }
}

export const getTestCheese = () => new Cheese(
    1,
    'Brie',
    'https://cheese.com/media/img/cheese/Brie_PDCo3RG.jpg',
    'Cream',
    30
);
